scripts/system/libraries/entitySelectionTool.js

```diff
@@ -1084,6 +1083,35 @@ SelectionDisplay = (function() {
         return tool;
     }
 
+    // toolHandle:  The overlayID associated with the tool
+    // that correlates to the tool you wish to query.
+    // @return: bool - Indicates if the activeTool is that queried.
+    function isActiveTool(toolHandle) {
+        var wantDebug = true;
+        if ( ! activeTool || ! toolHandle ){
+            if(wantDebug){
+                if ( activeTool ){
+                    print("WARNING: entitySelectionTool.isActiveTool - Encountered invalid toolHandle: " + toolHandle );
+                }
+            }
+            return false;
+        }
+        if ( ! grabberTools.hasOwnProperty( toolHandle ) ) {
+            if(wantDebug){
+                print("WARNING: entitySelectionTool.isActiveTool - Encountered unknown grabberToolHandle: " + toolHandle );
+            }
+            return false;
```

Comment:
Not sure you want this `return false`.  Even if it's not a recognized handle, it's best not to lie about identity.  I think you probably *do* want the warning here though (and if you remove this `return false;` you can merge the `if`s and avoid nesting).